automatic/kubernetes-cli/kubernetes-cli.nuspec

```diff
@@ -19,46 +19,81 @@
+
+Automatically places containers based on their resource requirements and other constraints, while not sacrificing
+availability. Mix critical and best-effort workloads in order to drive up utilization and save even more resources.
+
+#### [IPv4/IPv6 dual-stack](https://kubernetes.io/docs/concepts/services-networking/dual-stack)
 
-#### Service discovery and load balancing
-No need to modify your application to use an unfamiliar service discovery mechanism. Kubernetes gives containers their own IP addresses and a single DNS name for a set of containers, and can load-balance across them.
+Allocation of IPv4 and IPv6 addresses to Pods and Services.
 
-#### Automated rollouts and rollbacks
-Kubernetes progressively rolls out changes to your application or its configuration, while monitoring application health to ensure it doesn't kill all your instances at the same time. If something goes wrong, Kubernetes will rollback the change for you. Take advantage of a growing ecosystem of deployment solutions.
+#### [Self-healing](https://kubernetes.io/docs/concepts/workloads/controllers/replicationcontroller/#how-a-replicationcontroller-works)
 
-#### Secret and configuration management
-Deploy and update secrets and application configuration without rebuilding your image and without exposing secrets in your stack configuration.
+Restarts containers that fail, replaces and reschedules containers when nodes die, kills containers that don't respond
+to your user-defined health check, and doesn't advertise them to clients until they are ready to serve.
 
-#### Storage orchestration
-Automatically mount the storage system of your choice, whether from local storage, a public cloud provider such as GCP or AWS, or a network storage system such as NFS, iSCSI, Gluster, Ceph, Cinder, or Flocker.
+#### [Service discovery and load balancing](https://kubernetes.io/docs/concepts/services-networking/service)
+
+No need to modify your application to use an unfamiliar service discovery mechanism. Kubernetes gives Pods their own IP
+addresses and a single DNS name for a set of Pods, and can load-balance across them.
+
+#### [Secret and configuration management](https://kubernetes.io/docs/concepts/configuration/secret)
+
+Deploy and update secrets and application configuration without rebuilding your image and without exposing secrets in
+your stack configuration.
+
+#### [Batch execution](https://kubernetes.io/docs/concepts/workloads/controllers/job)
 
-#### Batch execution
 In addition to services, Kubernetes can manage your batch and CI workloads, replacing containers that fail, if desired.
+
+#### [Horizontal scaling](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale)
+
+Scale your application up and down with a simple command, with a UI, or automatically based on CPU usage.
+
+#### [Designed for extensibility](https://kubernetes.io/docs/concepts/extend-kubernetes)
+
+Add features to your Kubernetes cluster without changing upstream source code.
```

Comment:
The description here is too long.

Packages on the community repository can not have descriptions that are longer than 4000 characters, and according to my calculations, you have 4161 characters for the description.